library/packages/src/modules/PackageCallbacks.rb

```diff
@@ -286,9 +289,8 @@ def DoneProvide(error, reason, name)
             if @showLongInfo
               error_symbol = "ERROR"
 
-              if error == 1
-                error_symbol = "NOT_FOUND"
-              elsif error == 2
+              # https://github.com/openSUSE/libzypp/blob/8dda46306f06440e1acaefb36fb60f6ce909fd42/zypp/ZYppCallbacks.h#L106
```

Comment:
I do not get why you remove that 1 as it is still valid according to URL you paste

Reply:
and as bonus, using case again would be really nice, with default branch so we know that libzypp extend their enum.

Reply:
You only will get to this line when `error` is 2 or 3. The `1` case is handled through the MediaChange callback (see c9816a1cf7 and 546dcfb363). But I am fine with having it back.

Reply:
then please mention it also in comment, as link you paste indicate that also "1" should be handled, so explain why it is not needed.